cloudinary.js

```
@@ -0,0 +1,19 @@
+const cloudinary = require('cloudinary').v2
+const Multer = require('multer')
+
+cloudinary.config({
+  cloud_name: process.env.CLOUD_NAME,
+  api_key: process.env.API_KEY,
+  api_secret: process.env.API_SECRET
+})
+async function handleUpload(file) {
+  const res = await cloudinary.uploader.upload(file, {
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)